CONTRIBUTING.md

```diff
@@ -41,13 +50,19 @@ GitHub, clone, and develop on a branch. Steps:
    $ git commit
    ```
 
-   to record your changes in Git, then push the changes to your GitHub account with:
+5. Add a meaningful commit message. Pull requests are "squash-merged", e.g.
+   squashed into one commit with all commit messages combined. The commit
+   messages can be edited during the merge, but it helps if they are clearly
+   and briefly showing what has been done in the commit.   
+   
```

Comment:
I think an example commit message could be good here, and we could reference some guidelines, e.g. the 7 rules [here](https://www.theserverside.com/video/Follow-these-git-commit-message-guidelines) look quite good to me; many other conventions I looked up had more details about using "feat" or "fix" etc. and I don't think we need to be that formal.

Reply:
I also didn't want to add too much restrictions here, but I like the link you found - will add it, thanks.